app/services/file_manager/file_manifests.py

```diff
@@ -36,6 +36,7 @@ def __init__(self, interactive=True):
 
         self.interactive = interactive
         self.endpoint = self.app_config.Connections.url_bff + '/v1'
+        self.endpoint = 'http://localhost:5080/v1'
```

Comment:
why we have this localhost API endpoint here? For local testing?

Reply:
ah will remove it!